inductiva/_cli/cmd_logs/logs.py

```diff
@@ -27,9 +91,16 @@ def stream_task_logs(args):
 def register(parser):
     cli_utils.show_help_msg(parser)
 
-    parser.add_argument("task_id",
-                        type=str,
-                        help="ID of the task for which to consume the stream.")
+    parser.add_argument(
+        "mode",
+        type=str,
+        nargs="?",
+        default="SUBMITTED",
+        help=(
+            "Mode of log retrieval. "
+            "Use 'SUBMITTED[-n]' for the last submitted tasks, "
+            "'STARTED[-n]' for the last started tasks, "
+            "or a specific 'task_id' to retrieve logs for a particular task."))
```

Comment:
I'm a bit confused here. Is submitted the same as submitted-1 ? Do they both represent the last submitted task ?

Reply:
No. 
Submitted is the last submitted task
Submitted-1 is the second last submitted task.

This is similar behavior to git's one:
git show HEAD shows the last log
git show HEAD~1 shows the second last log

Reply:
Then we need a better explanation here. Maybe add some examples in the docs.

Reply:
Yes I agree! What would you suggest to improve this explanation?
What do you think about this:
```python
help=(
        "Mode of log retrieval. "
        "Use 'SUBMITTED' for the last submitted task, "
        "'SUBMITTED-1' for the second last submitted task, and so on. "
        "'STARTED' and 'STARTED-n' follow the same pattern for started tasks. "
        "Or, use a specific 'task_id' to retrieve logs for a particular task."
    )
```

Reply:
LGTM!